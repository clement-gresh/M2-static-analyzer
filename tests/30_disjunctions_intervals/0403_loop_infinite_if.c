{
  int x;
  x = rand(0,15);
  if(x < 10){
    while (x < 10) {
      print(x);
      x = x - 1;
    }
  }
  print(x);
}

