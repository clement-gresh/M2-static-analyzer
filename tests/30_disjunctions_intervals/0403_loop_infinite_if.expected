tests/30_disjunctions_intervals/0403_loop_infinite_if.c:6.6-15: { [ x in [0;9] ] }
tests/30_disjunctions_intervals/0403_loop_infinite_if.c:6.6-15: { [ x in [-∞;9] ] }
tests/30_disjunctions_intervals/0403_loop_infinite_if.c:10.2-11: { ⊥ }, { [ x in [10;15] ] }
analysis ended
